.github/workflows/containerd.yml

```diff
@@ -169,6 +169,13 @@ jobs:
           ginkgo version
           cp $(command -v ginkgo) /usr/local/bin
 
+      - name: Install cni on Linux
+        if: startsWith(matrix.os, 'ubuntu')
+        shell: bash
+        run: |
+          sudo -E PATH=$PATH script/setup/install-cni $(grep containernetworking/plugins go.mod | awk '{print $2}')
```

Comment:
might want to move this up to line 101 and do it right after installing containerd node working directory  
`${{ github.workspace }}/src/github.com/containerd/containerd`

Reply:
I wanted to keep is consistent with Windows. Perhaps both CNI installations can be moved to earlier stage - right after containerd installation

Reply:
kk..